q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -60,30 +76,125 @@ def decontam_score_viz(output_dir, decontam_scores: qiime2.Metadata,
     gray_lab_arr = []
     red_lab_arr = []
     blue_lab_arr = []
+    data_arr = []
+    display_seqs_arr = []
+    true_fasta_dest = []
+    contam_fasta_dest = []
+    nan_fasta_dest = []
+    sorted_key_arr = []
+    feature_or_read_arr = []
 
     for key in table_dict.keys():
         table = table_dict[key]
         decontam_scores = decontam_scores_dict[key]
-        df = decontam_scores.to_dataframe()
+        df = decontam_scores
```

Comment:
Why rename this? I think it would help with interpretability if you kept referring to it as `decontam_scores`.

Reply:
This was removed as suggested and the name was kept at decontam_scores

addressed in [8c3a5b5](https://github.com/qiime2/q2-quality-control/pull/82/commits/8c3a5b5ef158d988e3cac88141b30804175dd728)